site/src/Tour.jsx

```diff
@@ -1,6 +1,6 @@
+import PropTypes from "prop-types";
 import Joyride, { ACTIONS, EVENTS, LIFECYCLE } from "react-joyride";
 import { useState } from "react";
-import LayerIcon from "./icons/icon-layers.svg?react";
 import "./Tour.css";
 
 const Steps = [
```

Comment:
looks like you removed part of the tour and killed Step 9? It goes from 8/10 to 10/10. You can assign me a task to update the tour. 
<img width="687" height="387" alt="Screenshot 2025-08-08 at 6 09 22 PM" src="https://github.com/user-attachments/assets/b0b30947-0db8-4d48-92fa-aa9cff5fc70a" />
<img width="544" height="352" alt="Screenshot 2025-08-08 at 6 09 28 PM" src="https://github.com/user-attachments/assets/50b290b2-7150-4e3e-a25d-586022a37019" />